### PR TITLE
Fix global subcommand stdout and return type

### PR DESCRIPTION
**Pull request by falkecarlsen**

### CODE REVIEW

**Comment by MathiasMehl** [comment]
Looks good, but I think the #168 might create some merge conflict with this

**Comment by falkecarlsen** [comment]
Fix for the fix incoming

**Review by MathiasMehl — APPROVED** [review]
Looks good, #168 might create some merge conflicts, heads-up.

**Comment by falkecarlsen** [comment]
> Looks good, #168 might create some merge conflicts, heads-up.

You're right, it refactors it entirely. Merge is fixed

File: cli/src/main.rs [hunk 1]
```diff
@@ -189,16 +189,32 @@ fn main_inner() -> Result<(), String> {
...
+                if result {
+                    if !quiet_flag {
+                        println!("Model satisfies formula: {}", result);
+                    }
+                    std::process::exit(42);
+                } else {
+                    if !quiet_flag {
+                        println!("Model satisfies formula: {}", result);
+                    }
+                    std::process::exit(43);
```
Comment (NicEastvillage) [inline cli/src/main.rs:203]: Remove the duplicate code by moving the print outside the outer if.

File: cli/src/main.rs [hunk 2]
```diff
@@ -541,7 +557,15 @@ fn parse_arguments() -> ArgMatches<'static> {
...
                 .add_formula_arg()
-                .add_formula_type_arg(),
+                .add_formula_type_arg().arg(
+                Arg::with_name("quiet")
+                    .short("q")
+                    .takes_value(false)
+                    .long("quiet")
+                    .help(
+                        "Suppress stdout and only return exitcode 42 for true result or 43 for false",
+                    ),
```
Comment (NicEastvillage) [inline cli/src/main.rs:567]: Since this is now applies to two subcommands it should be moved to `cli::args`

File: cli/src/main.rs (hunk 1 above)
Comment (falkecarlsen) [reply cli/src/main.rs]: Fixed in e6e2663

File: cli/src/main.rs (hunk 2 above)
Comment (falkecarlsen) [reply cli/src/main.rs]: Fixed in e6e2663

**Pull request merged**